src/registrar/templates/includes/summary_item.html

```diff
@@ -142,6 +145,9 @@ <h4 class="margin-bottom-05">Invited domain managers</h4>
         </ul>
       {% endif %}
     
+    <!-- this is just to prevent the portfolio from appearing on Org Overview page -->
+    {% elif sub_header_text == "Organization Admins" %}
```

Comment:
This seems a bit hacky to me, though I don't understand what's happening here enough to offer a non-hacky solution. I'll offer a slightly better but still hacky suggestion below.